src/app/onboarding/final/final.page.ts

```diff
@@ -11,12 +12,18 @@ import { environment } from '@environments/environment';
 export class FinalPage implements OnInit, AfterViewInit {
   steps: number;
   templateType: string;
+  projectIcon: string;
+  onboardingData;
```

Comment:
we don't need to make it as a variable of this Class, since we are not using it in HTML file